### PR TITLE
linked icon to account/settings page

### PR DESCRIPTION
**Pull request by Regi-Rexus**

### CODE REVIEW

File: src/pages/home/index.tsx [hunk 1]
```diff
@@ -1,52 +1,66 @@
 import AddListCard from "@/components/AddListCard";
+import { FullHeader } from "@/components/FullHeader";
 import Head from "next/head";
-// import { UserCircleIcon } from "@heroicons/react/24/outline";
```
Comment (NoHara42) [inline src/pages/home/index.tsx:3]: please remove instead of commenting out here

File: src/pages/home/index.tsx [hunk 2]
```diff
@@ -1,52 +1,66 @@
...
-  return (
-    <>
-      <Head>
-        <title>Create Next App</title>
-        <meta name="description" content="Generated by create next app" />
-        <meta name="viewport" content="width=device-width, initial-scale=1" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-      <div className="w-[414]px h-[736]px flex flex-col">
-        <div className=" flex flex-row justify-between h-12 mt-4">
```
Comment (NoHara42) [inline src/pages/home/index.tsx:18]: this div can be removed

**Review by NoHara42 — CHANGES_REQUESTED** [review]
see changes :)

**Comment by Regi-Rexus** [comment]
sorry Ned, it was me who was using too big spacing! I changed it to 2 now

**Review by NoHara42 — APPROVED** [review]
LGTM

**Pull request merged**